evergreen.yml

```diff
@@ -536,15 +536,15 @@ functions:
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        remote_file: mongosql-odbc-driver/artifacts/${version_id}/windows-64/release/mongoodbc-signed.msi
+        remote_file: mongosql-odbc-driver/artifacts/${version_id}/windows-64/release/mongoodbc.msi
```

Comment:
Shouldn't this still be `mongoodbc-signed.msi`?
Because in `upload signed windows artifacts` we copy the file to `mongosql-odbc-driver/artifacts/${version_id}/windows-64/release/mongoodbc-signed.msi`